src/server/mod.rs

```diff
@@ -765,8 +765,6 @@ impl LanguageServer for LspServer {
                             let func = path.get(path.len() - 3)?;
                             match func {
                                 walk::Node::FunctionExpr(func) => {
-
-                                    // TODO Use MonoType::parameter directly
```

Comment:
`MonoType::parameter` is public now so you can actually resolve the TODO here by replacing the code below with a call to it.